src/tiledb/cloud/invites.py

```diff
@@ -0,0 +1,201 @@
+from tiledb.cloud import client
+from tiledb.cloud import rest_api
+from tiledb.cloud import tiledb_cloud_error
+from tiledb.cloud.rest_api.models.invitation_organization_join_email import (
+    InvitationOrganizationJoinEmail,
+)
+
+
+def accept_invitation(invitation_id: str) -> None:
+    """
+    Accept an invitation.
+
+    Args:
+        invitation_id (str): the ID of invitation about to be accepted.
+
+    Raises:
+        TileDBCloudError
+
+    Returns:
+        None
```

Comment:
We use Sphinx-style docstrings in here:

```
Accepts an invitation.

:param invitation_id: The UUID of the invitation to be accepted.
```

The `raises` section is probably not useful in this case.